service/src/main/java/uk/gov/hmcts/reform/fpl/service/CaseDataExtractionService.java

```diff
@@ -122,65 +122,36 @@ public CaseDataExtractionService(DateFormatterService dateFormatterService,
         return data.build();
     }
 
-    @SuppressWarnings("unchecked")
-    private Map<String, Object> prepareJudgeAndLegalAdvisor(JudgeAndLegalAdvisor judgeAndLegalAdvisor) {
-        ImmutableMap.Builder map = ImmutableMap.<String, Object>builder();
-
-        if (isEmpty(judgeAndLegalAdvisor)) {
-            return ImmutableMap.of(
-                "judgeTitle", EMPTY_PLACEHOLDER,
-                "legalAdvisorName", EMPTY_PLACEHOLDER
-            );
-        }
-
-        String judgeTitle = EMPTY_PLACEHOLDER;
-
-        if (isNotEmpty(judgeAndLegalAdvisor.getJudgeTitle())) {
-            judgeTitle = defaultIfBlank(judgeAndLegalAdvisor.getJudgeTitle().getLabel(), EMPTY_PLACEHOLDER);
-        }
-
-        map.put("judgeTitle", judgeTitle);
-
-        map.put("legalAdvisorName", defaultIfBlank(judgeAndLegalAdvisor.getLegalAdvisorName(), EMPTY_PLACEHOLDER));
-
-        if (isNotBlank(judgeAndLegalAdvisor.getJudgeLastName())) {
-            map.put("judgeLastName", judgeAndLegalAdvisor.getJudgeLastName());
-        }
-
-        if (isNotBlank(judgeAndLegalAdvisor.getJudgeFullName())) {
-            map.put("judgeFullName", judgeAndLegalAdvisor.getJudgeFullName());
-        }
-
-        return map.build();
-    }
-
     private Map<String, Object> getHearingBookingData(CaseData caseData) {
         if (caseData.getHearingDetails() == null || caseData.getHearingDetails().isEmpty()) {
-            return ImmutableMap.of(
-                "hearingDate", EMPTY_PLACEHOLDER,
-                "hearingVenue", EMPTY_PLACEHOLDER,
-                "preHearingAttendance", EMPTY_PLACEHOLDER,
-                "hearingTime", EMPTY_PLACEHOLDER,
-                "judgeName", EMPTY_PLACEHOLDER
-            );
+            return ImmutableMap.<String, Object>builder()
+                .put("hearingDate", EMPTY_PLACEHOLDER)
+                .put("hearingVenue", EMPTY_PLACEHOLDER)
+                .put("preHearingAttendance", EMPTY_PLACEHOLDER)
+                .put("hearingTime", EMPTY_PLACEHOLDER)
+                .put("hearingJudgeTitleAndName", EMPTY_PLACEHOLDER)
+                .put("hearingLegalAdvisorName", "")
+                .build();
```

Comment:
ImmutableMap.of() limited to maximum 5 key-value pairs... now 6 so have made the refactor